HW0/P6/P6B.py

```diff
@@ -10,22 +10,43 @@ def burnTime(t):
 if __name__ == '__main__':
     N = 16  # The number of jobs
     P = 4   # The number of processes
-
+    Time = 16
     # A thread pool of P processes
     pool = mp.Pool(P)
 
     # Use a variety of wait times
     ratio = []
-    wait_time = []
+    wait_time = [2e-6, 7e-6, 5e-5, 9e-5, 1e-4 ,6e-4 , 3e-3, 2e-2, 6e-1, 1 ]
 
     for t in wait_time:
         # Compute jobs serially and in parallel
         # Use time.time() to compute the elapsed time for each
         serialTime = 1
         parallelTime = 1
+        
+        #parallel time
+        pwake = time.time()
+        
+        results = pool.map(burnTime, [t]*Time)
+        
+        psleep = time.time()
+        parallelTime = pwake-psleep        
```

Comment:
You should take psleep-pwake (to get a positive number), although it shouldn't have an impact on the final result.